Makefile

```diff
@@ -543,6 +543,8 @@ clean-ui:
 	rm -rf webassets/*
 	rm -rf build.assets/.cache/ts
 	rm -rf web/packages/teleterm/build
+	# .pnpm-store is generated during dockerized builds.
+	rm -rf .pnpm-store
```

Comment:
Ok, nvm, I'm reverting this change that I added here.

Removing `.pnpm-store` is equivalent to running `go clean -modcache` and we don't want to run this on every build. I tried to make it so that this dir is created in a location in `/tmp` instead, but it's near impossible to do so (https://github.com/pnpm/pnpm/issues/7050#issuecomment-1998668286, https://github.com/pnpm/pnpm/issues/7100).

Instead, I made it so that `make -C build.assets clean` removes that dir.

Reply:
Ah, good call, thanks!